### DISCUSSION TITLE
Option to add "+X" placeholder for plugins listing people

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#1297] [Ideas]
An option to add a text mentioning the remaining people would be nice.

As an example: I have 83 followers right now and display them in my profile readme. This display is limited to 24 tho, resulting in only a fraction being listed while there would be more.
It could useful to have the last avatar being replaced by something like a circle containing text saying "+70", indicating there are 70 more...

**Comment by lowlighter** [comment]
Normally the number displayed above the profile picture should be correct one and you can set [`plugin_people_limit: 0`](https://github.com/lowlighter/metrics/blob/master/source/plugins/people/README.md#plugin_people_limit) if you want to display all pictures (or set an higher number)

I guess we could add a small text "And xxx more..." below the displayed icons, but I don't know if it'd look like a bit too cluttered
Maybe POC-ing it would help seeing if this is does look good

**Reply by Andre601** [reply]
True. The number is there. But you never know if someone can't read :P
